torchao/prototype/mx_formats/README.md

```diff
@@ -41,10 +41,21 @@ This is a module to do MX training, the MX matmul is currently emulated.
 
 ```python
 from torchao.prototype.mx_formats.mx_linear import swap_linear_with_mx_linear
-from torchao.prototype.mx_formats.config import MXLinearConfig
+from torchao.prototype.mx_formats.config import MXLinearConfig, MXGemmKernelChoice
+from torchao.utils import is_sm_at_least_100
+
+# early prototype: on MX-enabled hardware, you can use the real MX gemm backed by
+# torchao's CUTLASS kernels. In the future, we will also add cuBLAS kernel support.
+gemm_kernel_choice = MXGemmKernelChoice.EMULATED
```

Comment:
probably want just Emulated and "real" or whatever you want to call that

Reply:
I do want the soon-to-be-available cublas kernel to have a different setting compared to the cutlass kernel.

let's land now, this is just a prototype to unblock for now.  We can finalize the API naming later before broader release.